runtime/moonbase/src/lib.rs

```diff
@@ -638,7 +638,7 @@ parameter_types! {
 	pub const MinimumReward: Balance = 0;
 	pub const Initialized: bool = false;
 	pub const InitializationPayment: Perbill = Perbill::from_percent(30);
-	pub const MaxInitContributorsBatchSizes: u32 = 1000;
+	pub const MaxInitContributorsBatchSizes: u32 = 500;
```

Comment:
Is this to keep the PoV size small?

Reply:
No, its because with the new weights the 1000 contributors would exhaust the block resources

Reply:
We have a typescript test checking this and it failed with 1000, which is actually expected with the cost of the initialize function